yarn.lock

```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
 
 
+"@ampproject/remapping@^2.0.0":
```

Comment:
Same on my side 🤔

Reply:
@HichamELBSI @gu-stav when I do a `yarn setup` on master now it updates yarn.lock everytime since we merged it, shouldn't we update it for master branch once to avoid everyone having to do it separately?

Reply:
@ronronscelestes I talked to soupette about it: seems like we don't have a way to update a single dependency without updating all the others. In yarn@2 they have introduced `yarn up` for that, but we aren't using it by now.

Do you have a better idea?

Reply:
Oh I didn't know. Actually no I don't 😅
Just to be sure, does that mean that from now on we'll have to commit/push being careful not committing yarn.lock file? I just did it in my last PR because it caused tests failing without updating yarn.lock 🥲